src/data/archives.json

```diff
@@ -327,19 +327,23 @@
       {
         "name": "Funhaus 2020",
         "url": "https://archive.org/details/funhaus_2020"
-      }
+      },
+      {
+        "name": "Trash for Trash (Audio Podcast)",
+        "url": "https://archive.org/details/rt-trash-for-trash"
+      },
```

Comment:
Did this trailing comma cause it to fail?

Reply:
Good catch! Probably what made it fail yeah.

Reply:
This is what I get for editing in the browser 🤣